nfcli/parsers.py

```diff
@@ -10,6 +10,24 @@
 
 def get_content(content_data: dict) -> list[Content]:
     all_loads = []
+    
+    if content_data.get("StoredCraft"):
+        all_loads += content_data["StoredCraft"]["SavedStoredCraft"]
+        
+        craft_dict = {}
+        for load in all_loads:
+            if isinstance(load, dict):
+                if load["CraftTemplateKey"] in craft_dict:
+                    craft_dict[load["CraftTemplateKey"]] = craft_dict.get(load["CraftTemplateKey"]) + 1
+                else:
+                    craft_dict[load["CraftTemplateKey"]] = 1
+        
+        all_content = []
+        for craft in craft_dict:
+            all_content.append(Content(craft, craft_dict[craft]))
+        
+        return all_content
```

Comment:
Are we sure the carrier cannot have missiles / guns with ammunition? Are we sure we won't get regular ships with some small craft bays in the future? Even if no to both of these questions now, what about the future changes, and/or modded content? As such I'd avoid returning early here.